admin/categories.php

```diff
@@ -506,16 +506,16 @@ function init() {
           </div>
         </div>
         <div class="form-group">
-            <?php echo zen_draw_label(TEXT_EDIT_CATEGORIES_META_TAGS_KEYWORDS, '', 'class="col-sm-3 control-label"'); ?>
```

Comment:
Why remove the label, it is perfectly fine to use one, and adding the h3 changes the whole layout. of the page.

Reply:
The original label(s) was for multiple inputs: invalid html. There didn't appear to be a way to make it valid as labels should be per input and so I thought that probably the inputs should have a label each for accessibility?
Either the text/old label should be around the inputs as a legend/fieldset or be just text as simply as possible, as I have done with simple formatting as h3. It moves the text boxes to the left, but I didn't think that mattered really.